themes/ee/asset/javascript/src/cp/date_picker.js

```diff
@@ -683,7 +697,7 @@ EE.cp.datePicker = {
 			return
 		}
 
-		elements.on('focus', function() {
+		elements.off('focus').on('focus', function() {
```

Comment:
@Yulyaswan I'm slightly concerned with this, as it removes ALL `focus` listeners. I think we need to namespace the event here